docs/multi-agent-applications.md

```diff
@@ -0,0 +1,333 @@
+from pydantic_ai_examples.sql_gen import system_prompt
+
+# Multi-agent Applications
+
+There are roughly four levels of complexity when building applications with PydanticAI:
+
+1. Single agent workflows — what most of this documentation covers
```

Comment:
```suggestion
1. Single agent workflows — what most of the `pydantic_ai` documentation covers
```
Saying "this documentation" sounds to me like you mean this file, but I think you mean every file _except_ this file, right?